wdl-lint/tests/lints/disallowed-declaration-name/source.wdl

```diff
@@ -0,0 +1,55 @@
+version 1.2
+
+#@ except: MissingRequirements, SnakeCase, NonmatchingOutput
+task test_declaration_names {
+    meta {
+        description: "This is a test of disallowed declaration names"
+    }
+
+    parameter_meta {
+        fileInput: "Not OK - has type suffix"
+        gtfFile: "Not OK - has type prefix"
+        my_int: "Not OK - has type suffix"
+        stringValue: "Not OK - has type suffix"
+        booleanFlag: "Not OK - has type suffix"
+        floatNumber: "Not OK - has type suffix"
+        arrayData: "Not OK - has type suffix"
+        validName: "OK"
+        reference: "OK"
+        genome: "OK"
+        count: "OK"
+    }
+
+    # Move the directive here to disable InputSorting
```

Comment:
```suggestion
```
this comment won't make sense once this is merged into `main`. It only makes sense in context of this PR review, and we don't want comments like this to be merged in so they shouldn't be committed.